pkgs/by-name/ce/cert-viewer/package.nix

```diff
@@ -16,6 +17,15 @@ buildGoModule rec {
 
   vendorHash = "sha256-jNT04bYH5L/Zcfvel673zr2UJLayCO443tvBGZjrBZk=";
 
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    $out/bin/cert-viewer --help-man > cert-viewer.1
```

Comment:
To support cross-compilation, perhaps we can use the build time cert-viewer to emit the man page?

Reply:
PR: https://github.com/NixOS/nixpkgs/pull/306620